lib/auth/keystore/manager.go

```diff
@@ -183,18 +229,53 @@ func NewManager(ctx context.Context, cfg *servicecfg.KeystoreConfig, opts *Optio
 	}, nil
 }
 
+type sshCountSigner struct {
+	ssh.Signer
+}
```

Comment:
you wouldn't need a specific sshCountSigner (you could reuse cryptoCountSigner) if you wrapped the crypto signer right before the call to `ssh.NewSignerFromSigner`

Reply:
👍 updated in [1481a9c](https://github.com/gravitational/teleport/pull/45338/commits/1481a9c17f390c336d25b9d46730992137318b1c)